pkg/config/config.go

```diff
@@ -52,6 +52,8 @@ const (
 var (
 	// ConfigMapSourceDir represents a location where ConfigMap is attached to the pod
 	ConfigMapSourceDir = mountBaseDir + "/config-map"
+	// SysprepSourceDir represents a location where a Sysprep ConfigMap is attached to the pod
```

Comment:
Drop "ConfigMap" from this comment, it applies to both ConfigMap & Secret cases.
Same below in `SysprepDisksDir` comment.